index.html

```diff
@@ -246,6 +248,9 @@ <h2 class="contact-h2">I'm always interested in hearing about new projects, so i
       </li>
     </ul>
     <div class= "emailaddress">danielutibe07@gmail.com</div>
+    <div class="contactframe1"> <img src="./Images/contactframe1.svg" alt=""></div>
+    <div class="contactframe2"><img src="./Images/contactframe2.svg" alt=""></div>
+    <div class="contactframe3"> <img src="./Images/contactframe3.svg" alt=""></div>
     </section>
     <ul class="socials">
       <li>
```

Comment:
- [ ] to match the Figma design kindly align your social icons to the center, i suggest adding `align-items: center;` to your `.social` class 🙏 
![image](https://user-images.githubusercontent.com/89282221/214827062-003161da-0b54-49f7-b551-949f41ed1fcd.png)